packages/react/spec/auth/useSession.spec.ts

```diff
@@ -44,7 +45,6 @@ describe("useSession", () => {
             firstName
             googleImageUrl
             googleProfileId
-            id
```

Comment:
New versions of the clients sort these a bit differently, but its still deterministic.